rules/autoconsent/toyota.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "toyota",
+    "vendorUrl": "https://www.toyota.com/",
+    "cosmetic": false,
+    "runContext": {
+        "main": true,
+        "frame": false,
+        "urlPattern": ""
+    },
+    "prehideSelectors": ["#cookie-banner.visible"],
+    "detectCmp": [
+        {
+            "exists": "#cookie-banner.visible .cookie-banner__container .cookie-banner__accept"
```

Comment:
Selectors are quite generic, I wonder if there’s risk of side effects. Should we set `urlPattern` since it’s site specific?

Reply:
I think it's ok, worst case it will detect but fail to opt-out, and toyota might have other domains where they use this popup